### ISSUE TITLE
[BUG] Using RiskBudgeting optimization with min_weights other than the default throws error

### ISSUE THREAD

**Issue by vpmail** [skfolio/skfolio#96]
**Describe the bug**
Using RiskBudgeting optimization with min_weights other than the default throws the following error:
"The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()"

**To Reproduce**
I modified the Risk Parity - Variance Example slightly and it's not working.
```python
from plotly.io import show
from sklearn.model_selection import train_test_split

from skfolio import Population, RiskMeasure
from skfolio.datasets import load_sp500_dataset
from skfolio.optimization import InverseVolatility, RiskBudgeting
from skfolio.preprocessing import prices_to_returns

prices = load_sp500_dataset()

X = prices_to_returns(prices)
X_train, X_test = train_test_split(X, test_size=0.33, shuffle=False)

model = RiskBudgeting(
    risk_measure=RiskMeasure.VARIANCE,
    portfolio_params=dict(name="Risk Parity - Variance"),
    min_weights=np.array([0.02]*X_train.shape[1]) # <== Here!
)
model.fit(X_train)
model.weights_
```

**Expected behavior**
RiskBudgeting applies constraints without exception.

**Additional context**
The error message clearly describes the cause of the error.

**Versions**
0.5.0

Thank you,
Peter

**Comment by HugoDelatte** [comment]
Hello, you are correct, good spot!
This one slipped through the unit tests. I'm going to push a fix either tonight or tomorrow.

**Comment by HugoDelatte** [comment]
Hello, this is now fixed in v0.5.1.
